<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_TrybSieciowy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tryb sieciowy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TrybLokalny">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tryb lokalny</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsb_Dane">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-999999999999999983222784.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000000000008821361405306422640701865984.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Opis">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Otrzymane dane:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_WynikDane">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartLoop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Uruchom pętle</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Info">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_klie">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>połączeni klienci:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wartość: </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_taktowanie1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>T: jednostronne</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_taktowanie2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>T: dwustronne</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
